config/environments/production.rb

```diff
@@ -85,5 +85,5 @@
   config.active_record.dump_schema_after_migration = false
 
   # Google analytics
-    config.google_tag_manager_id = 'GTM-WVFLH9'
+    config.google_tag_manager_id = 'GTM-MPK3QX'
```

Comment:
Great! 🌞 Since you are here, can you please remove the extra indentation at the beginning of this line? (which was already there) ⛏